t/publish.t

```diff
@@ -37,6 +37,8 @@ package t::InMemoryCache {
         my ($self, $key, $subref) = @_;
         delete $self->cache->{"subscribe::$key"};
     }
+    
+    sub ping { return "PONG"; }
```

Comment:
how does this get used?

Reply:
As we added a `$writer->ping()` in the writer, and the tests actually uses `t::InMemoryCache` to mock what Redis does, we need to add this in.

Cause, when the test code runs, when it reach the `$writer->ping()`, it couldn't find any `ping` function in `t::InMemoryCache`, it will throws an error.